geode-core/src/main/java/org/apache/geode/internal/tcp/ClusterConnection.java

```diff
@@ -3006,7 +3047,7 @@ private void readMessage(ByteBuffer peerDataBuffer) {
       } catch (IOException ex) {
         // ignored
       }
-    } else /* (nioMessageType == END_CHUNKED_MSG_TYPE) */ {
+    } else /* (messageType == END_CHUNKED_MSG_TYPE) */ {
```

Comment:
looks dead, maybe there was `else if` once upon a time...

Reply:
I think that's just an informative comment.  It sets the context for understanding that block of code, so I don't think it should be removed.